docs/userguides/accounts.md

```diff
@@ -3,12 +3,29 @@
 Accounts in Ape come from [AccountAPI](../methoddocs/api.html#ape.api.accounts.AccountAPI) implementations (e.g. from plugins).
 There are typically two types of accounts:
 
-1. Test accounts
-2. Live network accounts
+1. CLI Reference Accounts
```

Comment:
This is the same thing as a live network account, so I am little confused here.
Perhaps we should limit this PR to only the README for now...